packages/go/stbernard/workspace/golang/build.go

```diff
@@ -33,15 +33,17 @@ import (
 // BuildMainPackages builds all main packages for a list of module paths
 func BuildMainPackages(workRoot string, modPaths []string, env environment.Environment) error {
 	var (
+		err      error
 		errs     []error
 		wg       sync.WaitGroup
 		mu       sync.Mutex
+		version  semver.Version
 		buildDir = filepath.Join(workRoot, "dist") + string(filepath.Separator)
 	)
 
-	version, err := git.ParseLatestVersionFromTags(workRoot, env)
-	if err != nil {
-		return fmt.Errorf("parse latest version from git tags: %w", err)
+	if version, err = git.ParseLatestVersionFromTags(workRoot, env); err != nil {
+		log.Errorf("Failed to parse version from git tags, falling back to environment variable: %v", err)
+		version = *semver.MustParse(env["VERSION"])
```

Comment:
Additionally, it might be good to set this env to a constant somewhere. I took a shortcut on the SB_LOG_LEVEL one and threw it in main, but it might not hurt to do a proper refactor to keep all our environment related code together, perhaps pushing it all into the environment package (and having `main()` run NewEnvironment for us)